op-node/p2p/cli/load_signer.go

```diff
@@ -0,0 +1,33 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/urfave/cli"
+
+	"github.com/ethereum-optimism/optimism/op-node/flags"
+	"github.com/ethereum-optimism/optimism/op-node/p2p"
+)
+
+// TODO: implement remote signer setup (config to authenticated endpoint)
```

Comment:
Perhaps we want to implement this in the `op-service` code so then it can be imported into multiple codebases. "Signer type" or something like that, where its the authenticated remote signer (uses `eth_signTransaction` with mTLS auth) and a local signer

Reply:
It's an old todo, not part of this PR. At least it's isolated now, should be easy to change once we integrate signers more deeply.

Reply:
Ah got it, i see now that most of this was restructuring